CONTRIBUTING.rst

```diff
@@ -61,20 +61,23 @@ Ready to contribute? Here's how to set up `eggplant` for local development.
           `a simple git branching model <https://gist.github.com/jbenet/ee6c9ac48068889b0912>`_.
           Read the guide to get a good introduction to Git workflows.
 
+e.g.::
+
+    $ git clone https://github.com/kbhff/eggplant
```

Comment:
Maybe should be `git clone git@github.com:USERNAME/eggplant.git` to reflect the intention of forking on Github before cloning?